---
layout: default
---
<div class="home">
  <div class="materials-wrap">




        
    <h2 class="module-header">1/31/2021 Announcement: First day of classes: Welcome to Spring 2022!</h2>
    <p>6.884 students: our the first round of sign ups for paper presentations covering September presentation slots are now up. View it <a href="/assets/6884PaperPresentationSlots.pdf">here</a>!
    </p>
      
      
    <p>Additional course websites:</p>
    <ul>
        <li><a href="https://stellar.mit.edu/S/course/6/fa19/6.884/index.html">MIT Stellar</a></li>
        <li><a href="https://piazza.com/class/jzvee3jyo6z28i">Piazza</a> (discussion forum)</li>
        <li><a href="https://learning-modules.mit.edu/gradebook/index.html?uuid=/course/6/fa19/6.884#assignments">Learning Modules</a> ((where you download problem set assignments and upload your solutions)</li>
    </ul>

    <h2 class="module-header">Course description</h2>
    <p>Welcome to an exploration of computational challenges in the design of human therapeutics.  We will explore computational methods in the design and analysis of therapeutics, including small molecules, biologics, cell based therapies, and synthetic biology approaches.   Lectures will present essential computational methods on molecular design and optimization drawing upon recent results in machine learning.  Classes will include presentations by students on recent research results related to the computational design of therapeutics and efficacy.   Problem sets will explore computational methods for therapeutic design.  Topics include protein design, antibody optimization, small molecule design and characterization, and the engineering of viruses and cell lines for therapeutic effect. Experts from industry will present on their views of the promise of computational approaches, what is working, and what is needed.
    </p>

    <p>As part of the subject students taking the graduate version  will use cloud resources to implement solutions to problems in therapeutic design, first in problem sets that span a carefully chosen set of tasks, and then in an independent project. You will be programming using Python 3 and TensorFlow 1.12 in Jupyter Notebooks on the Google Cloud, a nod to the importance of carefully documenting your work so it can be precisely reproduced by others.</p>


    <h2 class="module-header">Syllabus and schedule</h2>
    <table class="table">
        <th>Date</th><th>Type</th><th>Title</th><th>Speaker</th><th>Role</th><th>Affiliation</th><th>Presentation Date</th><th>Papers</th>
        <tr><td>02/03</td><td>Lecture</td><td>T-cell vaccines</td><td>David Gifford</td><td>Professor</td><td>MIT</td><td></td><td></td></tr>
        <tr><td>02/10</td><td>Invited Speaker</td><td>mRNA-LNP delivery platforms</td><td>Qiaobing Xu</td><td>Professor</td><td>Tufts University</td><td></td><td></td></tr>
        <tr><td>02/17</td><td>Invited Speaker</td><td>COVID vaccines and AAV delivery platforms</td><td>Dan Barouch</td><td>Professor</td><td>Harvard Medical School, Center for Virology and Vaccine Research</td><td></td><td></td></tr>
        <tr><td>02/24</td><td>Invited Speaker</td><td>Regenerative medicine for Type 1 Diabetes</td><td>Adrian Veres</td><td>Postdoctoral Fellow</td><td>Harvard University</td><td></td><td></td></tr>
        <tr><td>03/03</td><td>Invited Speaker</td><td>ML for small molecule therapeutics</td><td>Connor Coley</td><td>Professor</td><td>MIT</td><td></td><td></td></tr>
        <tr><td>03/10</td><td>Invited Speaker</td><td>Using computational methods to address druggability and drug discovery challenges</td><td>Jose Duca</td><td>Global Head of Computer-Aided Drug Discovery</td><td>Novartis</td><td></td><td></td></tr>
        <tr><td>03/17</td><td>Invited Speaker</td><td>ML-driven small molecule selection in drug discovery</td><td>Jeremy Jenkins</td><td>Head of Data Science in Chemical Biology & Therapeutics</td><td>Novartis</td><td></td><td></td></tr>
        <tr><td>03/24</td><td>Spring Break</td><td>Enjoy your break!</td><td></td><td></td><td></td><td></td><td></td></tr>
        <tr><td>03/31</td><td>Invited Speaker</td><td>Synthetic Biology Approaches to Disease Therapeutics</td><td>Ron Weiss</td><td>Professor</td><td>MTI</td><td></td><td></td></tr>
        <tr><td>04/07</td><td>Invited Speaker</td><td>Antibody CDR Design  Using High-Capacity Machine Learning</td><td>Stefan Ewert</td><td>Associate Director</td><td>Novartis</td><td></td><td></td></tr>
        <tr><td>04/14</td><td>Invited Speaker</td><td>T-cell therapeutics</td><td>Michael Birnbaum</td><td>Professor</td><td>MIT</td><td></td><td></td></tr>
        <tr><td>04/21</td><td>Invited Speaker</td><td>ML based reprogramming of gene expression</td><td>Brendan Frey</td><td>Founder & CEO</td><td>Deep Genomics</td><td></td><td></td></tr>
        <tr><td>04/28</td><td>TBD</td><td>TBD</td><td>TBD</td><td>TBD</td><td>TBD</td><td></td><td></td></tr>
        <tr><td>04/28</td><td>Invited Speaker</td><td>CRISPR and protein design</td><td>Han Altae-Tran</td><td>PhD student</td><td>Broad Institute of MIT and Harvard</td><td></td><td></td></tr>
    </table>

    <h2 class="module-header">Prerequisites</h2>
    <p>Undergraduate version:  Fundamental knowledge of machine learning, programming, and biology (GIR level). You should be comfortable with calculus, linear algebra, (Python) programming, probability, and introductory molecular biology.   Graduate version: Understanding of machine learning with Python and commonly-used libraries.  The graduate version is targeted towards students with a high degree of fluency in Computation and Biology and to fully understand the material, students will be best off having previously taken machine learning or computational systems biolog (6.874) or a similar course.</p>
    
    <h2 class="module-header">Class meeting times and places</h2>
    <ul>
        <li>Lecture: TR 11AM-12:30PM MIT Room 56-114</li>
        <li>Lecture: T 2PM-3PM MIT Room 26-328</li>
    </ul>

    <h2 class="module-header">Contact</h2>
</p><p>The best way to get detailed questions answered is to visit TA office hours or post them on <a href="https://piazza.com/class/jzvee3jyo6z28i">Piazza</a>.</p>

    <h2 class="module-header">Office hours</h2>
    <div class="materials-item">
        David Gifford (gifford@mit.edu): Office hours by appointment
    </div>
    <div class="materials-item">
       Hyunjin Park (hyunjinp@mit.edu): TBD, Stata Center, G5 Lounge
    </div>

    <h2 class="module-header">Grading</h2>
    <p><p>Class presentations (30%), programming-intensive problem sets (30%), and a final project (40%). Attendance in lecture is important as the class moves quickly and you will need to be present. There will be a requirment for  students to give presentations on relevant research papers from the syllabus list. If you will need special accomodations, please email Student Support Services - S3). We will be happy to accomodate!</p>

    <h2 class="module-header">Project</h2>
    <p>Final project details TBA</p>

    <h2 class="module-header">Papers</h2>
    <p>Lectures will be given by MIT professors and guests from industry on current topics at the forefront of research in computational therapeutics design. These lectures will be accompanied by academic papers listed in the course schedule and available through the <a href="https://stellar.mit.edu/S/course/6/fa19/6.884/index.html"> Stellar site</a></p>
  </div>
</div>
